UITests/Receipts/TransactionDetailTests.swift

```diff
@@ -22,11 +22,7 @@ class TransactionDetailTests: BaseTests {
         transactDetails.openReceipt(row: 0)
         let transactionDetailHeaderLabel = transactDetails.detailHeaderTitle
         waitForNonExistence(transactionDetailHeaderLabel)
-        if #available(iOS 12, *) {
-            verifyPayment(payment: "Payment\nMay 24, 2019", amount: "+6.00\n\(currency)")
-        } else {
-            verifyPayment(payment: "Payment May 24, 2019", amount: "+6.00 \(currency)")
-        }
+        verifyPayment(payment: "Payment\nMay 24, 2019", amount: "+6.00\n\(currency)")
```

Comment:
Should remove "+" after https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/71 merged